client/my-sites/customer-home/cards/tasks/site-setup-list/index.jsx

```diff
@@ -213,96 +211,54 @@ const SiteSetupList = ( {
-								className="site-setup-list__nav-back"
-								onClick={ () => setCurrentDrillLayoutView( 'nav' ) }
-							/>
-						) }
-						{ translate( 'Site setup' ) }
-					</>
-				</CardHeading>
+			{ useDrillLayout && <CardHeading>{ translate( 'Site setup' ) }</CardHeading> }
+			{ ! useDrillLayout && (
+				<CurrentTaskItem
+					currentTask={ currentTask }
+					skipTask={ () => skipTask( dispatch, currentTask, tasks, siteId, setIsLoading ) }
+					startTask={ () =>
+						startTask( dispatch, currentTask, siteId, advanceToNextIncompleteTask )
+					}
+					setTaskIsManuallySelected={ setTaskIsManuallySelected }
+				/>
 			) }
-			{ ( ! useDrillLayout || currentDrillLayoutView === 'nav' ) && (
-				<div className="site-setup-list__nav">
-					{ ! useDrillLayout && <CardHeading>{ translate( 'Site setup' ) }</CardHeading> }
-					{ tasks.map( ( task ) => {
-						const enhancedTask = getTask( task );
 
-						return (
+			<div className="site-setup-list__nav">
+				{ ! useDrillLayout && <CardHeading>{ translate( 'Site setup' ) }</CardHeading> }
+				{ tasks.map( ( task ) => {
+					const enhancedTask = getTask( task );
+					const isCurrent = task.id === currentTask.id;
+					const isCompleted = task.isCompleted;
+
+					return (
+						<>
 							<NavItem
 								key={ task.id }
 								taskId={ task.id }
 								text={ enhancedTask.label || enhancedTask.title }
-								isCompleted={ task.isCompleted }
-								isCurrent={ task.id === currentTask.id }
+								isCompleted={ isCompleted }
+								isCurrent={ isCurrent }
 								onClick={ () => {
 									setTaskIsManuallySelected( true );
 									setCurrentTaskId( task.id );
-									setCurrentDrillLayoutView( 'task' );
 								} }
-								showChevron={ useDrillLayout }
+								useDrillLayout={ useDrillLayout }
 							/>
-						);
-					} ) }
-				</div>
-			) }
-			{ ( ! useDrillLayout || currentDrillLayoutView === 'task' ) && (
-				<div className="site-setup-list__task task">
-					<div className="site-setup-list__task-text task__text">
-						{ currentTask.isCompleted ? (
-							<Badge type="info" className="site-setup-list__task-badge task__badge">
-								{ translate( 'Complete' ) }
-							</Badge>
-						) : (
-							<div className="site-setup-list__task-timing task__timing">
-								<Gridicon icon="time" size={ 18 } />
-								{ translate( '%d minute', '%d minutes', {
-									count: currentTask.timing,
-									args: [ currentTask.timing ],
-								} ) }
-							</div>
-						) }
-						<h2 className="site-setup-list__task-title task__title">{ currentTask.title }</h2>
-						<p className="site-setup-list__task-description task__description">
-							{ currentTask.description }
-						</p>
-						<div className="site-setup-list__task-actions task__actions">
-							{ currentTask.actionText && (
-								<Button
-									className="site-setup-list__task-action task__action"
-									primary
-									onClick={ () =>
+							{ useDrillLayout && isCurrent ? (
```

Comment:
I prefer to use && when there are multiple conditions and ternary for conditional rendering because it makes it explicit that what I want to render. I know that both are booleans but still this make code easier to understand. There are falsy values like 0 and NaN that react would render.